rpc/legacyrpc/methods.go

```diff
@@ -349,7 +349,7 @@ func addMultiSigAddress(icmd interface{}, w *wallet.Wallet) (interface{}, error)
 		return nil, err
 	}
 
-	return p2shAddr.EncodeAddress(), nil
+	return w.ChainParams().CashAddressPrefix + ":" + p2shAddr.EncodeAddress(), nil
```

Comment:
NIT: Maybe we want `EncodeAddressWithPrefix()`? Seems we are duplicating this code a lot...